docs/BR.md

```diff
@@ -1008,6 +1043,62 @@ CAs are permitted to treat a record lookup failure as permission to issue if:
+
+The response returned from each corroborating Network Perspective MUST allow the CA to verify:
+- a. the presence of the expected 1) Random Value, 2) Request Token, 3) IP Address, or 4) Contact Address, as required by the relied upon validation method specified in Sections 3.2.2.4 and 3.2.2.5; and
+- b. the CA's authority to issue to the requested domain(s), as specified in Section 3.2.2.8.
+
+[Section 3.2.2.4](#3224-validation-of-domain-authorization-or-control) and [Section 3.2.2.5](#3225-authentication-for-an-ip-address) describe the validation methods that require the use of Multi-Perspective Issuance Corroboration and how a Network Perspective can corroborate the outcomes determined by the Primary Network Perspective. A CA MAY use corroborating evidence for CAA record compliance for a period equal to the period of validity for Validation of Domain Authorization or Control specified in [Section 4.2.1](#421-performing-identification-and-authentication-functions).
+
+Results or information obtained from one Network Perspective MUST NOT be reused or cached when performing validation through subsequent Network Perspectives (e.g., different Network Perspectives cannot rely on a shared DNS cache to prevent an adversary with control of traffic from one Network Perspective from poisoning the DNS cache used by other Network Perspectives). The network infrastructure providing Internet connectivity to a Network Perspective MAY be administered by the same organization providing the computational services required to operate the Network Perspective. All communications between a remote Network Perspective and the CA MUST take place over an authenticated and encrypted channel relying on modern protocols (e.g., over HTTPS).
+
+CAs MAY immediately retry Multi-Perspective Issuance Corroboration using the same validation method or an alternative method (e.g., a CA can immediately retry validation using "Email to DNS TXT Contact" if "Agreed-Upon Change to Website - ACME" does not corroborate the outcome of Multi-Perspective Issuance Corroboration). When retrying Multi-Perspective Issuance Corroboration, CAs MUST NOT rely on corroborations from previous attempts. There is no stipulation regarding the maximum number of validation attempts that may be performed in any period of time.
+
+The "Quorum Requirements" Table describes quorum requirements related to Multi-Perspective Issuance Corroboration. Network Perspectives are considered distinct when the straight-line distance between the two States, Provinces, or Countries they reside in is at least 500 km. Network Perspectives are considered "remote" when they are distinct from the Primary Network Perspective and the other Network Perspectives represented in a quorum.
+
+Table: Quorum Requirements
+
+| # of Distinct Remote Network Perspectives Used | # of Allowed non-Corroborations |
+| --- | --- |
+| 2-5 |  1  |
+| 6+  |  2  |
+
+Remote Network Perspectives performing Multi-Perspective Issuance Corroboration:
```

Comment:
I couldn't find rule about where a remote perspective considered to be? Natural answer would be where is the metel it hosted is as this removed no delegation requirement we can't guarantee everything RP does happen at same place. For example HTTP validation RP in JP re‐delegate DNS lookup to public DNS hosted in US and still be legal; but can we say that RP is actually at anywhere?

Reply:
The definition of a network perspective's location is with the definition of a "Network Perspective" in the definitions section (line 383). I am curios if you would recommend putting this somewhere else in the document.

 https://github.com/ryancdickson/staging/blob/0144a4c0e544173d22a0f09fd4c45d4a52519992/docs/BR.md?plain=1#L383

The text on location currently reads:

`The location of a Network Perspective is determined by the point where unencapsulated outbound Internet traffic is first handed off to the network infrastructure providing Internet connectivity to that perspective.`

In many cases this equates to the definition you mentioned of having the location be where the metal hosting the perspective is. Notably in cases of VPN tunneling, the perspective's location becomes the location where VPN traffic is unencapsulated.

@orangepizza You do have a good point about not everything happening in one place, and I had thought a bit about that DNS angle when coming up with text for this line. Ultimately, I felt the most important attribute of this definition was for it to be well-defined and auditable, and given the fact the DNS is often outsourced, I felt it would be cleaner to define it as where the perspective itself hands off Internet traffic.

I think two points to note when considering this is that:
1) the only current diversity requirement in the document references the service regions of different RIRs (essentially continents), so some variance in location between a perspective and its DNS provider will not undermine that diversity
2) I personally find it unlikely for performance reasons that a perspective would outsource DNS to another continent. All of the major public DNS resolvers are globally anycast, and other 3rd-party DNS resolvers (e.g., from local ISPs) will also likely be relatively close to the perspective in question. It is usually faster to run DNS resolvers near where queries are originated, and I feel this will help prevent the problem you are discussing. DNS resolvers in other continents can also cause issues with goegraphic load balancing, so distant DNS resolvers are generally not recommended for that reason as well.

It is worth mentioning that at Princeton we have recently made significant advances in tooling to measure the effectiveness of deployed MPIC implementations against real-world ethical BGP attacks. These measurements include measurements of DNS behavior (i.e., we can determine whether a perspective's DNS was fooled by a BGP hijack). I feel that this type of work could shed light on whether outsourced DNS is a real problem for MPIC (in which case maybe a more-specific definition is needed) or if the system generally behaves as expected. In the meantime, I personally like the simpler definition as I do not want compliance and auditability to be obstacles for the MPIC initiative.

Reply:
Well, there is another rule(L1060) about network locations:  `Network Perspectives are considered distinct when the straight-line distance between the two States, Provinces, or Countries they reside in is at least 500 km`. and I don't think public resolver should be able to used in MRP context, as for CAA or some challenges like 3.2.2.4.7 or call to contact on DNS challenges DNS lookup is only thing that remote perspective will done, and "public resolver" isn't something really definable: only definition it can have without making actual whitelist will be "any open resolver is public DNS", and allowing that won't end well.

Reply:
To clarify, I simply mentioned public DNS resolvers as examples of DNS resolvers. I do not think it makes sense to add any specific text regarding public DNS resolvers.

I see your point that for CAA and some challenges remote perspectives are only sending DNS queries. What about adding text to ensure DNS resolver locations have the same diversity required of network perspectives? Potentially around line 1057 https://github.com/ryancdickson/staging/blob/0144a4c0e544173d22a0f09fd4c45d4a52519992/docs/BR.md?plain=1#L1057

I think a statement along these lines could enforce DNS resolver diversity.

`A network perspective MAY use a recursive DNS resolver that is not co-located with that network perspective. However, the DNS resolver used by a network perspective MUST fall within the same Regional Internet Registry service region as that network perspective. Furthermore, for any pair of DNS resolvers used on a Multi-Perspective Issuance Corroboration attempt, the straight-line distance between the two States, Provinces, or Countries the DNS resolvers reside in MUST be at least 500 km. The location of a DNS resolver is determined by the point where unencapsulated outbound DNS queries are first handed off to the network infrastructure providing Internet connectivity to that DNS resolver.`

Reply:
Minor tweaks, addressed in https://github.com/ryancdickson/staging/commit/6cca62b0b96ee49295e1d7c61cf5f85b7a1a5f27.